package.json

```diff
@@ -240,7 +241,6 @@
     "jest": "^29.5.0",
     "jest-canvas-mock": "^2.4.0",
     "jest-environment-jsdom": "^29.5.0",
-    "jest-junit": "^16.0.0",
```

Comment:
I think this will break junit reports for frontend unit tests 🤔

Reply:
IDK exactly what that does, but I was thinking since we were switching to a different reporter I wasn't sure if `jest-junit` was still needed. Easy enough to put the `junit-jest` changes back in.